Update usage.md for air-gapped & disconnected updates (#245)
CentOS/centos-bootc: docs/usage.md

RUN systemctl mask bootc-fetch-apply-updates.timer
```

> This is useful for environments where manually updating the systems is

Reviewer: Undefined "this"s require me to think more than I want :^)

```suggestion
Disabling the updates timer is useful for environments where manually updating the systems is
```

COPY 50-weekly.conf /usr/lib/systemd/system/bootc-fetch-apply-updates.timer.d
```

## Air-gapped and dissconnected updates

For environments without a direct connection to a centralized container

> registry, we encourage mirroring a registry in DMZ if possible or manually

Reviewer: I am not entirely sure how to interpret DMZ.  Should we point readers to /etc/containers/registries.conf [1] instead?

[1] https://www.redhat.com/sysadmin/manage-container-registries

moving container images using `skopeo copy`.

> Systems that are completely dissconnected and require manual updates via USB

Reviewer: ```suggestion
Systems that are completely disconnected and require manual updates via USB
```

> drives this is relatively simple.

Reviewer: The "this" is undefined.  Should we rephrase the sentence to something like:

> For systems that are completely disconnected and require manual updates via USB drives, you may also use Skopeo.

Copy image to USB Drive:

```skopeo copy docker://[registry]/[path to image] dir://run/media/$USER/$DRIVE/$DIR```

> *note, Using the dir transport will create a number of files,

Reviewer: ```suggestion
NOTE: Using the `dir` transport will create a number of files,
```

Should we point the user to https://github.com/containers/image/blob/main/docs/containers-transports.5.md?

> and it's recommended to place the image in it's own directory.

Reviewer 2: ```suggestion
and it's recommended to place the image in its own directory.
```

i.e. s/it's/its/

If the image is local the containers-storage transport will transfer
the image from a system directly to the drive:

```skopeo copy containers-storage:[image]:[tag] dir://run/media/$USER/$DRIVE/$DIR```

From the client system, insert the USB drive and mount it:

```mount /dev/$DRIVE /mnt```

`bootc switch` will direct the system to look at this mount point for future
updates, and is only necessary to run one time if you wish to continue
consuming updates from USB devices. note that if the mount point changes,
simply run this command to point to the alternate location. We recommend
using the same location each time to simplfy this.

> ```bootc switch --transport dir --retain /mnt/$DIR```

Reviewer 2: Hmm, why `--retain`?  I'm thinking we should actually drop that option because the direction lately is to make the source of truth the bootloader entries.

Author: That's only from me misinterpreting the output of -h. Let's remove it!
